apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -468,14 +586,15 @@ static void tc_task_vfork(void)
  ****************************************************************************/
 int task_main(void)
 {
-	tc_task_task_create();
-	tc_task_task_delete();
-	tc_task_task_restart();
-	tc_task_exit();
 	tc_task_atexit();
+	tc_task_exit();
+	tc_task_getpid();
 	tc_task_on_exit();
 	tc_task_prctl();
-	tc_task_getpid();
+	tc_task_task_create();
+	tc_task_task_delete();
+	tc_task_task_reparent();
```

Comment:
task_reparent is not a public api. I think we don't need to make not public api test case

Reply:
ok, then for code coverage what should we do ?

Reply:
only calling with public api, we can cover all internal apis.